setup.cfg

```diff
@@ -20,22 +20,17 @@ classifiers =
   Topic :: Software Development
   Topic :: Scientific/Engineering
   Operating System :: Microsoft :: Windows
-  Operating System :: POSIX
-  Operating System :: Unix
+  Operating System :: POSIX :: Linux
   Operating System :: MacOS
   Programming Language :: Python
-  Programming Language :: Python :: 3.7
-  Programming Language :: Python :: 3.8
-  Programming Language :: Python :: 3.9
```

Comment:
why remove this? It's relatively little effort to update this once per release, that is: add new versions we support, and remove old ones :thinking:

Reply:
See my response above

Reply:
yes I prefer to be explicit here